include/deal.II/fe/fe_interface_values.h

```diff
@@ -427,6 +742,24 @@ class FEInterfaceValues
                       const unsigned int q_point,
                       const unsigned int component = 0) const;
 
+  /**
+   * Create a view of the current FEInterfaceValues object that represents a
+   * particular scalar component of the possibly vector-valued finite element.
+   * The concept of views is explained in the documentation of the namespace
+   * FEInterfaceViews.
```

Comment:
```suggestion
   * FEValuesViews
```
We actually don't explain anything here...